test/test_mutant_protein_sequence.py

```diff
@@ -67,6 +70,8 @@ def test_mutant_amino_acids_in_mm10_chrX_8125624_refC_altA_pS460I():
         min_alt_rna_reads=1,
         min_variant_sequence_coverage=1,
         variant_sequence_assembly=True)
+    core_logic.process_variants()
```

Comment:
Can we make this more "functional" by having `process_variants` just return its results? Or is there other persistent data that gets used later?